bot.py

```diff
@@ -281,6 +286,41 @@ def status_handler(update: Update, context: CallbackContext):
     )
 
 
+def broadcast_handler(update: Update, context: CallbackContext):
+    """"For moderators only"""
+    logging.debug(f"Entering: broadcast_handler, chat_id: {update.message.chat_id}, MODERATORS: {MODERATORS}")
+    if str(update.message.chat_id) not in MODERATORS:
+        logging.warning(f"{update.message.chat_id} tried to send an Broadcast, but is no Moderator")
+        context.bot.send_message(
+            update.message.chat_id,
+            f"Du hast nicht die Berechtigung einen Broadcast zu versenden. {emojize(error_emoji())}"
+        )
+        return None
+    text = demojize(" ".join(context.args))
+    if not text:
+        context.bot.send_message(
+            update.message.chat_id,
+            f"Broadcast-Text darf nicht leer sein. {emojize(error_emoji())}"
+        )
+        return None
+    emojized_text = emojize(text)
+    for user_id in user_db.users():
+        if user_id == update.message.chat_id:
+            logging.debug(f"Skipped {user_id}, text: {text}")
+            continue
+        logging.debug(f"Send to {user_id}, text: {text}")
+        try:
+            context.bot.send_message(user_id, emojized_text)
+        except Unauthorized:
+            logging.exception(f"Skipped {user_id}, because he blocked the bot")
+            continue
+    context.bot.send_message(
+        update.message.chat_id,
+        "Broadcast erfolgreich versendet."
```

Comment:
Maybe add a thumbs up emoji or something cool and creative here

Reply:
Yeah, good idea. Why am I always so boring? ^^